.github/actions/setup-msvc/action.yaml

```
@@ -0,0 +1,34 @@
+  export-path-to-vcvarsall:
+    description: 'The environment variable to store the path to vcvarsall.bat in'
+    required: false
+  export-path-to-vs:
+    description: 'The environment variable to store the path to the Visual Studio installation in'
+    required: false
+runs:
+  using: 'node20'
+  main: 'dist/index.js'
+branding:
+  icon: terminal
+  color: gray-dark
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)